dataLayer Builder w-Standard eCommerce.tpl

```diff
@@ -0,0 +1,295 @@
+    "displayName": "",
+    "id": "brand_dummy"
+  },
+  "containerContexts": [
+    "WEB"
+  ]
+}
+
+
+___TEMPLATE_PARAMETERS___
+
+[
+  {
+    "displayName": "Custom Parameters",
+    "name": "customParameters",
+    "simpleTableColumns": [
+      {
+        "defaultValue": "",
+        "displayName": "key",
+        "name": "name",
+        "type": "TEXT"
+      },
+      {
+        "defaultValue": "",
+        "displayName": "value",
+        "name": "value",
+        "type": "TEXT"
+      }
+    ],
+    "type": "SIMPLE_TABLE"
+  },
+  {
+    "displayName": "Transaction Parameters",
+    "name": "transactionDropdown",
+    "groupStyle": "ZIPPY_CLOSED",
+    "type": "GROUP",
+    "subParams": [
+      {
+        "displayName": "",
+        "name": "standardParameters",
+        "simpleTableColumns": [
+          {
+            "selectItems": [
+              {
+                "displayValue": "transactionId",
+                "value": "transactionId"
+              },
+              {
+                "displayValue": "transactionAffiliation",
+                "value": "transactionAffiliation"
+              },
+              {
+                "displayValue": "transactionTotal",
+                "value": "transactionTotal"
+              },
+              {
+                "displayValue": "transactionTax",
+                "value": "transactionTax"
+              },
+              {
+                "displayValue": "transactionShipping",
+                "value": "transactionShipping"
+              },
+              {
+                "displayValue": "transactionProducts",
+                "value": "transactionProducts"
+              }
+            ],
+            "defaultValue": "",
+            "displayName": "key",
+            "name": "name",
+            "type": "SELECT"
+          },
+          {
+            "defaultValue": "",
+            "displayName": "value",
+            "name": "value",
+            "type": "TEXT"
+          }
+        ],
+        "type": "SIMPLE_TABLE"
+      }
+    ]
+  }
+]
```

Comment:
Funny you say this, but I actually had this in my first version, but removed to keep the template simpler, but do agree, this would force people to have to set an event to trigger events and tags when it fires. I have added this field into the template as requested.